src/controller/ExamplePersistentStorage.h

```diff
@@ -36,8 +37,6 @@ class PersistentStorage : public chip::PersistentStorageDelegate
      * null, falls back to getenv("TMPDIR") and if that is not set falls back
      * to /tmp.
      *
-     * If non-null values are provided, the memory they point to is expected to
-     * outlive this object.
      */
     CHIP_ERROR Init(const char * name = nullptr, const char * directory = nullptr);
```

Comment:
This may be a good opportunity to change the signature to const std::string for both name and directory?